packages/api-explorer/src/components/DocSDKs/DocSDKs.tsx

```diff
@@ -51,6 +54,10 @@ interface LanguageSDKProps {
   type?: IType
 }
 
+const StyledHeading = styled(Heading)`
+  color: ${({ theme }) => theme.colors.text3};
+`
```

Comment:
Prefer to see use of `Heading variant` here if possible (admittedly none map to `text3`)

https://github.com/looker-open-source/components/blob/master/packages/components/src/Text/text_variant.ts

Reply:
Without a closely matching variant, I think it's best to use the text3 color for now. These color choices have more to do with the visual hierarchy so I'd hate to lose that unless the implementation here is just not advisable.

Reply:
Fair enough. Can we get a more semantic name in place in that case so developers know when/why to reach for this component (also helps give @looker/components a signal about possibly missing variants) :)

Reply:
Updated to Headig3 per our conversation on Slack.